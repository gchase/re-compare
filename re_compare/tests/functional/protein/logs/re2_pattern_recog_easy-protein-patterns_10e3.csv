regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,0.05793571472167969,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,0.05793571472167969,"[-1, -1]"
DHKNLD.,first_match,0,0.027179718017578125,"[-1, -1]"
DHKNLD.,all_matches,0,0.027179718017578125,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,0,0.05817413330078125,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,0,0.05817413330078125,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,0.04982948303222656,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,0.04982948303222656,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.05793571472167969,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.05793571472167969,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.006198883056640625,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.06413459777832031,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,0,0.09226799011230469,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,0,0.09226799011230469,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.3199577331542969,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.3199577331542969,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.0209808349609375,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.3409385681152344,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,0,0.06413459777832031,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,0,0.06413459777832031,"[-1, -1]"
P..L...KWE.,first_match,0,0.05316734313964844,"[-1, -1]"
P..L...KWE.,all_matches,0,0.05316734313964844,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,0.06389617919921875,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,0.06389617919921875,"[-1, -1]"
