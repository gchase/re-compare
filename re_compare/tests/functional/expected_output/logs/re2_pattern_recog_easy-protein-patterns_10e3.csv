regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,0.054836273193359375,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,0.054836273193359375,"[-1, -1]"
DHKNLD.,first_match,0,0.04792213439941406,"[-1, -1]"
DHKNLD.,all_matches,0,0.04792213439941406,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,0,0.05078315734863281,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,0,0.05078315734863281,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,0.037670135498046875,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,0.037670135498046875,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.06031990051269531,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.06031990051269531,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.006198883056640625,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.06651878356933594,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,0,0.08821487426757812,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,0,0.08821487426757812,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.23508071899414062,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.23508071899414062,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.013828277587890625,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.24890899658203125,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,0,0.04982948303222656,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,0,0.04982948303222656,"[-1, -1]"
P..L...KWE.,first_match,0,0.04100799560546875,"[-1, -1]"
P..L...KWE.,all_matches,0,0.04100799560546875,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,0.23818016052246094,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,0.23818016052246094,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),first_match,1,0.12493133544921875,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,1,0.12493133544921875,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),all_matches,1,0.13375282287597656,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),first_match,0,0.05316734313964844,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),all_matches,0,0.05316734313964844,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),first_match,0,0.030040740966796875,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),all_matches,0,0.030040740966796875,"[-1, -1]"
A|G....G,first_match,1,0.14090538024902344,"[15, 15]"
A|G....G,consecutive,1,0.14090538024902344,"[15, 15]"
A|G....G,consecutive,2,0.008106231689453125,"[19, 19]"
A|G....G,all_matches,2,0.14901161193847656,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,first_match,0,0.06079673767089844,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,all_matches,0,0.06079673767089844,"[-1, -1]"
